dev/benchmarks/complex_layout/lib/drawer.dart

```diff
@@ -0,0 +1,91 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/scheduler.dart' show timeDilation;
+import 'package:flutter/widgets.dart';
+
+import 'main.dart';
```

Comment:
Given that these two files import each other, maybe we should just combine all the code into one file.